### DISCUSSION TITLE
override the render method of BlocksView

### DISCUSSION THREAD

**Discussion by momu-2016** [GrapesJS/grapesjs#5223] [Q&A]
Hello, I'm creating my own plugin and I need to customize the layout of blocks. For example, I want to put categories in one <div> and blocks in another <div>. Essentially, I want to completely change the layout of the block section. How can I override the render method of BlocksView in my plugin to achieve custom styling?

**Comment by artf** [comment]
You can [replace the Block Manager with your own UI](https://grapesjs.com/docs/modules/Blocks.html#customization)
